custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -286,8 +286,8 @@ class Outcome(models.Model):
     @property
     def hiv_status(self):
         return {
-            None: None,
-            'NULL': None,
+            None: 'unknown',
```

Comment:
are there lots of these in the data? it's going to cause everyone to show up on the "HIV not updated" task lists, which is maybe fine/good.